doc/newsboat.asciidoc

```diff
@@ -1038,6 +1038,7 @@ sequence identifier `[char]` contains any non-whitespace characters, then
 `[format 1]` will be evaluated and inserted, otherwise `[format 2]` will be
 evaluated and inserted. The `&` and `[format 2]` are optional, i.e. if the
 identifier's text is empty, then an empty string will be inserted.
+Newsboat also provides a way center a string in a given width and with any given whitespace character with `%=[char][width]`. If width it = 0 the whole width of the window will be used.
```

Comment:
We changed the format, so please update the documentation here to match.

Reply:
Is updated and broken down into two smaller sentences instead of one huge one.